pkg/jetstream/generated/informers/externalversions/factory.go

```diff
@@ -39,6 +39,7 @@ type sharedInformerFactory struct {
 	lock             sync.Mutex
 	defaultResync    time.Duration
 	customResync     map[reflect.Type]time.Duration
+	transform        cache.TransformFunc
```

Comment:
@vsinger The "deny purge" specific changes look good. Can you share what this addition does and how it is used below?

Reply:
@bruth this has been generated after the local build. I think the files have not been generated since the latest dependency updates. The `transform` func has been added to `kubernetes/client-go`  a while ago, see here: https://github.com/kubernetes/client-go/commit/411a1189f8f9319ed91062da062eda5857ebc790

Reply:
Understood thanks!